src/app/nation/nation.component.html

```diff
@@ -0,0 +1,53 @@
+
+  <h1>Add Nation </h1>
+  <form class="form-horizontal"(ngSubmit)="onSubmit(nation.value)" #nation="ngForm">
+    <div class="form-group">
+      <label for="admin_name">Admin Name</label>
+      <input type="text" class="form-control" id="admin_name" required minlength="5" maxlength="30" [(ngModel)]="nation.admin_name" name="admin_name" #admin_name="ngModel">
+      <div *ngIf="admin_name.errors && (admin_name.dirty || admin_name.touched)" class="alert alert-danger" id = "danger">
```

Comment:
The `id` referenced here is not unique, which is invalid HTML.  Please use a `class` for this.

Also, the space around the equals should be removed.